WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/StatsFragment.kt

```diff
@@ -109,6 +109,12 @@ class StatsFragment : DaggerFragment() {
                 }
             }
         })
+
+        viewModel.selectedDateChanged.observe(this, Observer { statsGranularity ->
```

Comment:
Why do wee need to propagate the change back to the Fragment? I understand why we want all events to go through the VM, but this is the opposite case.

Btw do we ever call `SelectedDateProvider.selectDate`?

Reply:
We need to go through the fragment in order for the LiveData to work. 

> Btw do we ever call SelectedDateProvider.selectDate?

Not yet, I'll have another PR with the Overview block which will call the `selectDate`